spec/rails_helper.rb

```diff
@@ -53,6 +31,16 @@
     DatabaseCleaner.strategy =
       example.metadata[:js] ? :truncation : :transaction
     DatabaseCleaner.start
+OmniAuth.config.test_mode = true
+OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+  provider: "facebook",
+  uid: "123456",
+  info:
+  { name: "Blessing Ebowe",
+    email: "blessingebowe@gmail.com",
+    profile_img: "http://graph.facebook.com/1065771400114300/picture"
+  }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.